### PR TITLE
upgrade_packages

### PR DESCRIPTION
**Pull request by brandon-binary**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "@babel/register": "^7.0.0",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.2",
-    "babel-plugin-inline-react-svg": "^1.0.0",
+    "babel-plugin-inline-react-svg": "^1.1.0",
```
Comment (morteza-binary) [inline package.json]: @brandon-binary How about other vulnerability packages

File: package.json [hunk 2]
```diff
@@ -133,5 +133,8 @@
     "react": "^16.4.0",
     "react-dom": "^16.4.0",
     "react-transition-group": "^2.5.0"
+  },
+  "resolutions": {
+    "js-yaml": "^3.13.1"
```
Comment (morteza-binary) [inline package.json]: @brandon-binary would you please specify which package is dependent on `js-yaml`?

Comment (brandon-binary) [reply package.json]: `babel-plugin-inline-react-svg`

File: package.json (hunk 1 above)
Comment (brandon-binary) [reply package.json]: There shouldn't be other vulnerable packages after upgrading this.

Comment (morteza-binary) [reply package.json]: @brandon-binary After applying your changes in my local machine, `yarn audit` gave me 508 vulnerabilities packages 
image

Comment (brandon-binary) [reply package.json]: Did you reinstall npm_module?

**Pull request closed without merging**